### PR TITLE
Added documentation for resizing managed disks.

### PR DESCRIPTION
**Pull request by BaileyLawson**

### CODE REVIEW

**Comment by suehighmoor** [comment]
Not reviewed the content yet, but looks like you need to change the actual file name to reflect the metadata

**Comment by cblackuk** [comment]
> 
> 
> Not reviewed the content yet, but looks like you need to change the actual file name to reflect the metadata

100% correct

**Comment by BaileyLawson** [comment]
> Not reviewed the content yet, but looks like you need to change the actual file name to reflect the metadata

My bad, missed that. Fixed.

**Comment by suehighmoor** [comment]
Do we still need the original file for resizing unmanaged disks? Or does this article replace that one?

**Comment by BaileyLawson** [comment]
> Do we still need the original file for resizing unmanaged disks? Or does this article replace that one?

The original is incorporated in this article, so this acts as a replacement.

File: articles/azure/azs-how-resize-os-drive.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ---
-title: How to resize an unmanaged disk - PowerShell | UKCloud Ltd
-description: Provides help for resizing an unmanaged disk on UKCloud for Microsoft Azure
+title: How to resize an OS drive | UKCloud Ltd
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-os-drive.md]: title metadata tag should match the main article heading, so change this to

How to resize an OS drive using PowerShell

File: articles/azure/azs-how-resize-os-drive.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-toc_title: Resize an unmanaged disk - PowerShell
-toc_fullpath: Users/How To/azs-how-resize-unmanaged-disk.md
-toc_mdlink: azs-how-resize-unmanaged-disk.md
+toc_title: How to resize an OS drive
+toc_fullpath: Users/How To/azs-how-resize-os-drive.md
+toc_mdlink: azs-how-resize-os-drive.md
 ---
 
-# How to resize an unmanaged disk - PowerShell
+# How to resize an OS drive with PowerShell
```
Comment (suehighmoor) [inline articles/azure/azs-how-resize-os-drive.md]: How to resize an OS drive using PowerShell

File: articles/azure/azs-how-resize-os-drive.md (hunk 1 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-resize-os-drive.md]: Fixed

File: articles/azure/azs-how-resize-os-drive.md (hunk 2 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-resize-os-drive.md]: Fixed

File: articles/azure/azs-how-resize-os-drive.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 author: Bailey Lawson
 toc_rootlink: Users
 toc_sub1: How To
 toc_sub2:
 toc_sub3:
 toc_sub4:
-toc_title: Resize an unmanaged disk - PowerShell
-toc_fullpath: Users/How To/azs-how-resize-unmanaged-disk.md
-toc_mdlink: azs-how-resize-unmanaged-disk.md
+toc_title: How to resize an OS drive
```
[the remark this answers is not in the document]
Comment (BaileyLawson) [reply articles/azure/azs-how-resize-os-drive.md]: Fixed. The title metadata does match the first heading, doesn't it? I may misunderstand your comment but I think it is correct now?

**Pull request merged**